app/views/organizations/adoptable_pets/index.html.erb

```diff
@@ -12,6 +12,50 @@
 <!--pet tiles-->
 <section class="pb-5" id="adoptable_pets">
   <div class="container">
+    <div class="row">
+      <div class="col m-2">
+      <!--filter section-->
+        <%= search_form_for @q, url:adoptable_pets_path do |f| %>
+          <div class='d-flex justify-content-center align-items-center flex-wrap mb-3'>
+            <div class="d-flex gap-3 flex-wrap mb-3">
+              <div class="form-group">
+                <%= f.label :name_i_cont, "Name" %>
+                <%= f.text_field :name_i_cont, class: "form-control", placeholder: "Enter a few characters" %>
+              </div>
+              <div class="form-group">
+                <%= f.label :sex_eq, "Sex" %>
+                <%= f.select :sex_eq, ["Male", "Female"], {include_blank: 'All'}, class: "form-select" %>
+              </div>
+              <div class="form-group">
+                <%= f.label :species_eq, "Species" %>
+                <%= f.select :species_eq, Pet.species, {include_blank: 'All'}, class: "form-select" %>
+              </div>              
+              <div class="form-group">
+                <%= f.label :breed_eq, "Breed" %>
+                <%= f.select :breed_eq, get_animals, {include_blank: 'All'}, class: "form-select" %>
+              </div>
+              <div class="form-group">
+                <%= f.label :ransack_birth_date, "Age" %>
+                <%= f.select :ransack_birth_date, age_ranges, {include_blank: 'All'}, class: "form-select"%>
+              </div>
+              <div class="form-group">
+                <%= f.label :ransack_adopted, "Status" %>
+                <%= f.select :ransack_adopted, [['Adopted', true], ['Unadopted', false]], {include_blank: 'All'}, class: "form-select" %>
+              </div>
```

Comment:
We can remove this filter option. All the pets on this page should be unadopted as we don't want to show publicly the info of pets that are adopted.